specification/source/glossary.rst

```diff
@@ -0,0 +1,26 @@
+.. _sec_glossary:
+
+Glossary of Terms
+----------------------

```

Comment:
Underlines should be the same length as the title, otherwise sphinxdoc complains